pass_import.py

```diff
@@ -32,6 +32,8 @@
 
 __version__ = '2.4'
 
+TMP_ATTACHMENT_DIR = "tmp_attachment_files/"
```

Comment:
This should not be static. You should use a secure temporary directory of the user with https://docs.python.org/3/library/tempfile.html. And therefore, it does not have to be a global variable.